tests/xmap_test.py

```diff
@@ -1251,5 +1251,85 @@ def testLoopCollectives(self):
       fm(x)
 
 
+class NamedAutodiffTests(jtu.JaxTestCase):
+
+  def test_vjp_reduce_axes(self):
```

Comment:
It's a minor thing, but so far in this file we've been using camel case for test names, so it would be good to keep it consistent